docs/src/components/Search/SearchModal.tsx

```diff
@@ -0,0 +1,211 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { useStaticQuery, graphql, navigate } from "gatsby";
+import capitalize from "capitalize";
+import { filter } from "fuzzaldrin-plus";
+import { useCombobox } from "downshift";
+import {
+  Portal,
+  Modal,
+  ModalHeader,
+  ModalSection,
+  Text,
+  mediaQueries,
+  useMediaQuery,
+} from "@kiwicom/orbit-components";
+import { Search as SearchIcon, ChevronRight } from "@kiwicom/orbit-components/icons";
+import { disableBodyScroll, clearAllBodyScrollLocks } from "body-scroll-lock";
+
+import StyledInputContainer from "./primitives/StyledInputContainer";
+import StyledPrefix from "./primitives/StyledPrefix";
+import StyledInput from "./primitives/StyledInput";
+import { StyledMenu, StyledMenuItem, StyledMenuItemTitle } from "./primitives/StyledMenu";
+
+interface Props {
+  onClose: () => void;
+}
+
+interface QueryResponse {
+  allMdx: {
+    nodes: Array<{
+      fields: {
+        slug: string;
+      };
+      frontmatter: {
+        description: string;
+      };
+    }>;
+  };
+}
+
+interface SearchResult {
+  name: string;
+  breadcrumbs: string[];
+  description: string;
+  path: string;
+}
+
+const StyledModalWrapper = styled.div`
+  /* align modal to the top */
+  ${mediaQueries.largeMobile(css`
+    > * > * {
+      align-items: flex-start;
+    }
+  `)}
+`;
+const StyledSearchWrapper = styled.div`
+  margin-bottom: 1rem;
+  font-size: 1rem;
+`;
+const StyledSearchInput = styled(StyledInput)`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
these styled don't have any effect on input, i suppose your intention was to truncate the text, but it does not, behaves in a same way with and without :)